hazelcast-code-generator/src/main/java/com/hazelcast/client/protocol/generator/CodeGenerationUtils.java

```diff
@@ -0,0 +1,15 @@
+package com.hazelcast.client.protocol.generator;
+
+public final class CodeGenerationUtils {
+
+    private CodeGenerationUtils() {
+    }
+
+    public static String capitalizeFirstLetter(String input) {
+        return input.substring(0, 1).toUpperCase() + input.substring(1);
```

Comment:
this depends on a default locale of JVM. 
is it what you want? perhaps you want to use a well-defined (fixed) locale.